code/modules/clothing/suits/armor.dm

```diff
@@ -126,18 +126,6 @@
 /obj/item/clothing/suit/storage/greatcoat/ironhammer
 	icon_state = "greatcoat_ironhammer"
 
-/obj/item/clothing/suit/storage/greatcoat/serbian_overcoat
```

Comment:
Unrefactors my refactor that united all greatcoats and overcoats as the "Greatcoat" type, which encompasses armoured and protective jackets

Unless there's a good reason for it, this should be undone